frontend/src/app/modules/bookings/bookings-page/bookings-page.component.sass

```diff
@@ -148,11 +129,11 @@ p, span
 
 @media only screen and (max-width: 1200px)
     .bookings-container
-        padding: 130px
+        padding: 100px
 
 @media only screen and (max-width: 800px)
```

Comment:
Why two media queries on lines 134 and 130 have same values? One media query can be removed